meteor/client/ui/DndListItemWrapper.tsx

```diff
@@ -0,0 +1,87 @@
+import { DragDropItemTypes } from './DragDropItemTypes'
+import React, { useRef } from 'react'
+import { DragSourceMonitor, DropTargetMonitor, useDrag, useDrop, XYCoord } from 'react-dnd'
+import { Identifier } from 'dnd-core'
+import { faGripLines } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+interface IDndItemWrapperProps {
+	index: number
+	dndType: DragDropItemTypes
+	listItem: any
+	className: string
```

Comment:
Do we really need to parse CSS classes as props? 
Ideally this wrapper only provided the drag and drop functionality and didn't have any say in the styling. The item should already know how it is styled.

Also, normally when using className you would expect the class to be applied to the entire component that you are writing className on, but here you parse it to a part of the component. This may or may not lead to bugs in the future.

Reply:
I don't see how else to manage the fact that the existing table style needs to be passed on to the `<td>` that holds it, while the left one, holding the drag-handle, has another style. 
I do agree that the name should be changed to indicate that this is what it does!

Reply:
We change the HTML structure to not use tables